spyder/widgets/findinfiles.py

```diff
@@ -259,7 +259,8 @@ def __init__(self, parent, search_text, search_text_regexp, search_path,
         # Layout 3
         hlayout3 = QHBoxLayout()
 
-        self.global_path_search = QRadioButton(_("Path"), self)
+        self.global_path_search = QRadioButton(_("Current Working "
+                                                 "Directory"), self)

```

Comment:
Please change this string to `Current working directory` (i.e. loose the capitals in `Working` and `Directory`)